src/Infrastructure/Doctrine/Repository/ArticleRepository.php

```diff
@@ -260,4 +287,27 @@ private function createQueryBuilder(array $filters, array $sortBy = [], array $s
 
         return $queryBuilder;
     }
+
+    private function joinDimensionContent(QueryBuilder $queryBuilder, string $alias): void
```

Comment:
we should move dimension based part to the `dimensionContentQueryEnhancer` service